src/Core/Form/IdentifiableObject/DataHandler/ManufacturerFormDataHandler.php

```diff
@@ -68,6 +68,13 @@ public function create(array $data)
             $data['shop_association'] = [];
         }
 
+        /** @var UploadedFile $uploadedLogo */
+        $uploadedLogo = $data['logo'];
+
+        if ($uploadedLogo instanceof UploadedFile) {
```

Comment:
Should we add the validation in `update(array $data)` too?

Reply:
No need, it's handled inside `ImageUploader::upload`